test/NuGet.Core.FuncTests/Dotnet.Integration.Test/MsbuildIntegrationTestFixture.cs

```diff
@@ -492,6 +492,12 @@ private void CopyRestoreArtifacts(string artifactsDirectory, string pathToSdkInC
                         overwrite: true);
                 }
             }
+
+            // temp: delete once the .NET SDK ships Newtonsoft.Json 13.0.1 or higher
```

Comment:
tracking issue?

Reply:
https://github.com/NuGet/Home/issues/11135

I'm not going to spin another build just to change the comment.